.github/workflows/pipeline.yml

```diff
@@ -88,7 +88,7 @@ jobs:
             - name: Apply changesets
               id: changesetVersion
               run: |
-                  echo ::set-output name=changes::$(pnpm ci:version 2>&1 | grep -q 'No unreleased changesets found' && echo 'false' || echo 'true')
+                  echo "changes=$(pnpm ci:version 2>&1 | grep -q 'No unreleased changesets found' && echo 'false' || echo 'true')" >> $GITHUB_OUTPUT
```

Comment:
not 100% sure about this one

Reply:
I have no idea :D

Reply:
@kranthie-sap would you know ?

Reply:
Hi @drktfl and @nlunets: This named variable `changes` from the output is used in the workflow [here](https://github.com/SAP/open-ux-odata/pull/440/files#diff-fe700949ba86403864aa7b6fdcab2c00c55e1047f73a6a282135a35e8a21e03bL94) and [here](https://github.com/SAP/open-ux-odata/pull/440/files#diff-fe700949ba86403864aa7b6fdcab2c00c55e1047f73a6a282135a35e8a21e03bL108). From the docs linked in the PR description, it does look like the right replacement for `set-output`. The only way to test if this works as expected is to verify the workflow runs after merging this and the subsequent release commit.